modules/core/src/views/orbit-view.js

```diff
@@ -1,120 +1,98 @@
 import View from './view';
 import Viewport from '../viewports/viewport';
 
-// TODO - use math.gl
-import {createMat4, transformVector} from '../utils/math-utils';
-import * as mat4 from 'gl-matrix/mat4';
+import {Matrix4} from 'math.gl';
 import OrbitController from '../controllers/orbit-controller';
 
 const DEGREES_TO_RADIANS = Math.PI / 180;
 
-// TODO - remove need for custom project overrides
-class OrbitViewport extends Viewport {
-  project(xyz, {topLeft = false} = {}) {
-    const v = transformVector(this.pixelProjectionMatrix, [...xyz, 1]);
-    const [x, y, z] = v;
-    const y2 = topLeft ? this.height - y : y;
-    return [x, y2, z];
-  }
+function getViewMatrix({height, fovy, orbitAxis, rotationX, rotationOrbit, zoom}) {
+  // Distance from camera to target that projects common space 1 unit = 1 screen pixel
+  const distance = 0.5 / Math.tan((fovy * DEGREES_TO_RADIANS) / 2);
 
-  unproject(xyz, {topLeft = false} = {}) {
-    const [x, y, z] = xyz;
-    const y2 = topLeft ? this.height - y : y;
-    return transformVector(this.pixelUnprojectionMatrix, [x, y2, z, 1]);
-  }
-}
+  const viewMatrix = new Matrix4().lookAt({eye: [0, 0, distance]});
 
-export default class OrbitView extends View {
-  // Get camera `distance` to make view fit a box centered at lookat position in the viewport.
-  // @param {Array} boundingBox - [sizeX, sizeY, sizeZ]], defines the dimensions of bounding box
-  static getDistance({boundingBox, fov}) {
-    const halfMaxSide = Math.max(boundingBox[0], boundingBox[1], boundingBox[2]) / 2;
-    const distance = halfMaxSide / Math.tan(((fov / 180) * Math.PI) / 2);
-    return distance;
-  }
+  const projectionScale = 1 / (height || 1);
```

Comment:
I will add more comments to this section.

We position the camera so that one common space unit (world space unit scaled by zoom) at the viewport center maps to one screen pixel. This is a similar technique to that used in web mercator projection. By doing so we are able to convert between common space and screen space sizes efficiently in the vertex shader (e.g. clamping `radiusMinPixels`, screen space extrusion, etc.).

The participating factors in achieving this are the fov, the aspect ratio and the height of the viewport. When `height` increases, we need to increase the distance from the camera to the target to keep the 1:1 mapping. However, this also changes the projected depth of each position. We have two ways of mitigating this:

1. Scale `distance`, `near` and `far` by `height`
2. Use constant `distance` and scale down the common space

This PR uses option 2 because both the camera distance and the common space are internal to the implementation of this view.